addOns/automation/CHANGELOG.md

```diff
@@ -4,7 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-
+### Fixed
+- Wrong AuthCheckingStrategy for request
```

Comment:
This should be plain English, e.g. "Use correct authentication verification method for `request`".